### ISSUE TITLE
Missing Ant Build Output on macOS Mojave

### ISSUE THREAD

**Issue by JoelProminic** [prominic/Moonshine-IDE#663]
I was working with @JustinProminic on a Java Gradle project with an Ant script on macOS Mojave.

When he ran the Ant script, Moonshine did not show the full command output.   

On the first test, the Ant script failed, and we could see the last few lines reporting the error, but not the output for the error itself.  It may be that this means that Moonshine is showing STDERR output only.

Once we fixed the problem, Moonshine showed nothing except the command when running the Ant script.  When we ran the same command in Terminal, the output showed correctly.

Ant scripts work fine for me on OSX Yosemite.  I'll try High Sierra when I have a chance.

**Comment by JoelProminic** [comment]
I tried this today on High Sierra, and the Ant output looked fine.  This seems like a new problem for Mojave.

**Comment by rat-moonshine** [comment]
The above problem tested with sandbox or non-sandbox build (?)

While I tested with stable v2.5.0 (non-sandbox) against sample Flex desktop project, running its bundled Ant script activities reported properly in console. I tested running Ant script in following ways:

1. Navigate into the 'build' directory, locate the Ant build XML file, right-click and chose 'Run Ant Script'
2. By `Others -> Build Apache Ant File` from top-down menu

Is there any available project with Ant build script available which I can download and test without doing much Ant adjustment (?)

**Comment by rat-moonshine** [comment]
We noticed a few problem during our investigations:

1. Moonshine do not notify on Ant build process ends/running (except console output, but output can halt during certain Ant process and create confusion)
2. Losing output of any running Ant process if another build requested before completing the former

Both the cases now addressed with following resolutions:

- Run-progress notification will shown at bottom-right status bar during process running
- A new build request before completing a running one will be ignored giving an alert

**Comment by rat-moonshine** [comment]
During my Windows' tests today I noticed Ant build process failed by giving following error on console:
"The syntax of the command is incorrect."

I found the local environment-setup with command execution also fails, when ran on standalone command line. Upon some more investigation, it appears that Windows export path command do not likes path surrounded with double-quotes, i.e. `set JAVA_HOME="C:\MoonshineSDKs\Java\openjdk-1.8.0"`. The way found problematic for many other exports like Ant, Flex etc. 

[EnvironmentSetupUtils](https://github.com/prominic/Moonshine-IDE/blob/master/ide/MoonshineDESKTOPevolved/src/actionScripts/utils/EnvironmentSetupUtils.as) now adjusted to export the paths without quote on Windows, which fix the problem.
